fuzzers/aflcc/fuzzer.py

```diff
@@ -0,0 +1,344 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration code for AFL fuzzer."""
+
+import shutil
+import os
+import threading
+import subprocess
+
+from fuzzers import utils
+
+from fuzzers.afl import fuzzer as afl_fuzzer
+
+def is_benchmark(name):
+    """ Check if the benchmark contains the string 'name' """
+
+    # TODO: return name in os.environ['BENCHMARK]
+    # return benchmark is not None and name in benchmark
+
+    # OSS-fuzz benchmarks do not have a consistent folder content
+    # but typically have the name of the benchmark in the PWD
+    pwd_dir = os.getenv("PWD", None)
+    if pwd_dir is None:
+        raise ValueError('PWD not defined')
+
+    if name in pwd_dir:
+        return True
+
+    # non-OSS-fuzz benchmarks
```

Comment:
nit: being comments with capital letters and end with periods please.

Reply:
done